.github/workflows/publish-helm-chart.yml

```
@@ -0,0 +1,244 @@
+name: Publish Helm Chart to AWS Public ECR
+
+on:
+  push:
+    branches:
+      - main
+      - memetics19:ft/helm
```

actionlint (1.7.12)

[error] character ':' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)